### PR TITLE
 Added QuadgramAssocMeasures

### LINKED ISSUE #2100: nltk.collocations.QuadgramAssocMeasures() is missing
nltk.collocations only import BigramAssocMeasures and TrigramAssocMeasures from nltk.metrics, and QuadgramAssocMeasures is left out.

### PR DESCRIPTION
**Pull request by bharatr21**
Fix #2100 by importing `QuadgramAssocMeasures` in `collocations.py` and `__init__.py` in metrics

### CODE REVIEW

**Comment by alvations** [comment]
Thanks @Bharat123rox! Forgot about this after closing #2101 without merge. 

I've added a doctest for the quadgram in https://github.com/nltk/nltk/blob/develop/nltk/test/metrics.doctest and https://github.com/nltk/nltk/blob/develop/nltk/test/collocations.doctest

LGTM once the CI test passes.

**Review by alvations — APPROVED** [review]

**Comment by stevenbird** [comment]
Thanks @Bharat123rox

**Pull request merged**
